FrontEnd/src/components/LoginScreen.js

```diff
@@ -63,6 +63,8 @@ function LoginScreen(props) {
                             style={{ height: "40%", borderColor: 'gray', borderWidth: 1, borderRadius: 5, width: "100%", marginBottom: "3%" }}
                             placeholder = "Nutzername"
                             onChange={handleUsernameChange}
+                            autoFocus={true}
+                            onSubmitEditing = {handleSubmit}
```

Comment:
Ist das das gewünschte verhalten? Wenn ich den Username eingebe und dann enter drücke, dass dann mit den entsprechenden credentials (potentiell leerses PW) ein loginversuch gestartet wird?

Reply:
Ich bin tatsächlich schon oft über die Situation gestolpert dass ich username und pw eingetragen habe --> enter --> falsche Anmeldedaten und dann der username falsch war. Dann korrigiert man den username, drückt enter und nix passiert. Man muss dann zusätzlich noch was klicken (maus oder TAB is egal) um sich einzuloggen.
Das onSubmitEditing beim username kann also potentiell ein klick sparen. Wenn versucht wird sich mit einem leeren PW anzumelden passiert einfach nix, außer die Meldung falsche Anmeldedaten. Also ich würdes ganz gern drinnelassen, hänge aber auch nicht mega dran. Darfst du entscheiden